### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][JAWS/NVDA][MAS4.2.1][IntelliJ][Wipro]: Jaws and NVDA are not annoucing the message when list is empty in Push image window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1501]
**Repro Steps:**

1. Create a new dynamic web project.
2. Navigate to and select Project ->Azure->Push Image.
3. Push image dialog will open.
4. Navigate to  "List" which is present under Before launch:Build Artifacts,Activate Tool window then observe the NVDA and JAWS Behavior

**Expected Result:**
**NVDA Behavior:**
NVDA should announce "There are no tasks to run before launch"

**Actual Result:**
**NVDA Behavior:**
NVDA is announcing as list
**Jaws Behavior:**
List view
To move to items use the Arrow keys.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![nvda is not announcing about list is empty and also message in push image](https://user-images.githubusercontent.com/22956130/36139667-82cd0ef6-10c4-11e8-87a7-3446ddf1766d.jpg)
![push image-list view](https://user-images.githubusercontent.com/22956130/36139721-af15a810-10c4-11e8-85a0-edff1b45f291.jpg)

**Comment by Eskibear** [comment]
Content hierarchy is provided by IntelliJ run/debug configuration, out of the toolkit's control.

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749941
